Cargo.toml

```diff
@@ -13,18 +13,18 @@ keywords = ["auth", "authorization", "rbac", "acl", "abac"]
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-regex = "1.3.9"
-rhai = { version = "0.19.0", features = ["sync", "only_i32", "no_function", "no_float", "no_optimize", "no_module", "no_closure", "serde", "unchecked"] }
+regex = "1.4.0"
+rhai = { version = "0.19.0", features = ["sync", "only_i32", "no_function", "no_float", "no_optimize", "serde", "unchecked"] }
```

Comment:
why do we remove `no_module` and `no_closure`? I thought it's not useful for our use case

Reply:
`no_function` also means `no_closure`. In addition, I cannot compile rhai 0.19.6 locally if I enable both `no_function` and `no_module`.

Reply:
@PsiACE yes rhai has some upstream issues

Reply:
we can simply wait for rhai 0.19.7

Reply:
Alright, I'll put out `0.19.7` within today or tomorrow...  I've included a CI test which will make sure it compiles for your combination of features...

Reply:
`0.19.7` released to `crates.io`.

Reply:
@schungx thanks! @PsiACE could you upgrade rhai?

Reply:
@GopherJ Done.